lib/aux_file.ml

```diff
@@ -30,12 +30,14 @@ let mk_absolute vfile =
 
 let start_aux_file ~aux_file:output_file ~v_file =
   let vfile = mk_absolute v_file in
-  oc := Some (open_out output_file);
-  Printf.fprintf (Option.get !oc) "COQAUX%d %s %s\n"
-    version (Digest.to_hex (Digest.file vfile)) vfile
+  try
+    oc := Some (open_out output_file);
+    Printf.fprintf (Option.get !oc) "COQAUX%d %s %s\n"
+      version (Digest.to_hex (Digest.file vfile)) vfile
+  with Sys_error _ -> ()
```

Comment:
Wouldn't it be better to print a warning instead of doing nothing?

Reply:
In general I'd say yes, but in this particular case I believe it is best this way for two reasons:
- if the user is really using a snap package, he can't do much to fix the warning (other than granting the snap full power, which is defeating the whole containment idea)
- if the file system is read only, for example, then the user is going to get a good error message (I hope) from the code trying to write the .vo file